Article/Article.js

```diff
@@ -3,18 +3,24 @@
 class Article {
   constructor(domElement) {
     // assign this.domElement to the passed in domElement
-    this.domElement;
+    this.domElement = domElement;
+
+    console.log(this.domElement);
     // create a reference to the ".expandButton" class. 
-    this.expandButton;
+    this.expandButton = domElement.querySelector('.expandButton');
     // Using your expandButton reference, update the text on your expandButton to say "expand"
-    
+    //expandButton.textContent = 
+    this.expandButton.textContent = 'expand';
+
     // Set a click handler on the expandButton reference, calling the expandArticle method.
-    
+    this.expandButton.addEventListener("click", e => {
```

Comment:
```suggestion
    this.expandButton.addEventListener("click", () => {
```
If you're not using the event object in your callback function, no need to pass it in.